feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -238,13 +294,15 @@ def main():
 
     fcEW = split_rectangle(
         lon0=-20., lon1=25., lat0=-80., lat1=-55., name='Eastern Weddell Sea',
```

Comment:
```suggestion
    fcEW = split_polygon(
        lon_vector=[-20., -20., 10., 25., 25.], 
        lat_vector=[-80., -60., -55., -55., -80.], 
        name='Eastern Weddell Sea',
```

Reply:
yes, that's exactly where I'd have put it, at 10E. Thanks @cbegeman!

Reply:
@cbegeman, if you give me the go-ahead, I'll commit the change, then remake the geojsons and rebase again.

Reply:
@xylar We're good to go! Thanks

Reply:
For reference, the Weddell gyre figure above is Fig. 2 from Orsi et al. 1993 ("On the circulation and stratification of the Weddell Gyre").

Reply:
@milenaveneziani Thanks! Reference added as a comment in the file